README.md

```diff
@@ -100,7 +100,7 @@ On an `aarch64` platform:
 ```bash
 > cd rust-vmm-dev-container
 > ./docker.sh build
-> ./docker.sh push
+> ./docker.sh publish
```

Comment:
I would mention in the commit message about this fix as well.

Reply:
done